ecpy_hqc_legacy/instruments/drivers/dll/alazar935x.py

```diff
@@ -98,9 +98,8 @@ def configure_board(self):
 
         board.setExternalTrigger(ats.DC_COUPLING, ats.ETR_5V)
 
-        triggerDelay_sec = 0.
-        triggerDelay_samples = int(triggerDelay_sec * self.samples_per_sec +
-                                   0.5)
+        triggerDelay_sec = 0
```

Comment:
Please avoid camelCase like name !

Reply:
in general I agree, but this is to keep as close as possible to the original python code from AlzarTech